res/map/AWBWMapConverter.py

```diff
@@ -0,0 +1,211 @@
+
+import sys
+import math
+import re
+import csv
+
+
+def convertFile(infile,outfile):
+	reader = csv.reader(infile)
+	# Fun fact: this used to write out to the file once for every loop iteration. That was... not exactly what you'd call efficient. This is... more so.
```

Comment:
While that <i>is</i> a fun fact, I'm not sure we need to preserve it for posterity.